script-class.js

```diff
@@ -16,3 +16,29 @@ document.querySelector('#addBookButton').addEventListener('click', () => {
   title.value = '';
   author.value = '';
 });
+
+function displayPage(link) {
```

Comment:
- [ ] Good job implementing the three required sections 👍 however according to the requirements you should Display the current date below the navigation bar. For this requirement, you should use a library - [Luxon](https://moment.github.io/luxon/#/install). See the article [Get the current date and time](https://moment.github.io/luxon/#/tour?id=get-the-current-date-and-time).